src/components/AuthoritiesLookup/AuthoritiesLookup.test.js

```diff
@@ -76,35 +77,38 @@ describe('Given AuthoritiesLookup', () => {
       fireEvent.click(linkStyleBtnOfRow);
     });
 
-    it('should hide the list of items (not unmount)', () => {
-      expect(screen.getByTestId('authority-search-results-pane')).toHaveAttribute('class', 'pane focusIndicator hidden');
+    it('should hide the list of items', () => {
+      expect(screen.queryByTestId('authority-search-results-pane')).not.toBeInTheDocument();
     });
 
     it('should open the detail view', () => {
       expect(screen.getByTestId('marc-view-pane')).toBeVisible();
     });
+
+    it('should pass correct data about the position of the selected list item', () => {
+      expect(MultiColumnList).toHaveBeenLastCalledWith(expect.objectContaining({
+        itemToView: {
+          selector: 'any',
+          localClientTop: 123,
```

Comment:
Please add a comment here to point where this `123` value is mocked. It will be confusing to find a random value and not know where it comes from

Reply:
Thanks, fixed